Geometry/HcalCommonData/python/caloSimulationParameters_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+from Geometry.HcalCommonData.caloSimulationParameters_cfi import *
+
+from Configuration.Eras.Modifier_dd4hep_cff import dd4hep
+
+dd4hep.toModify(caloSimulationParameters,
```

Comment:
@bsunanda where is this modifier supposed to be used?

Reply:
This is an advance statement. Hope we shall use this once we have a full dd4hep application.